packages/devtools-launchpad/src/components/Sidebar.js

```diff
@@ -19,6 +21,35 @@ const Sidebar = React.createClass({
     onSideBarItemClick: React.PropTypes.func.isRequired
   },
 
+  renderTitle(title) {
+
+    return dom.div({className: "title-wrapper"},
+                    dom.h1({}, "Debugger"),
+                    dom.div({className: "launchpad-container"},
+                            dom.div({className: "launchpad-container-icon", dangerouslySetInnerHTML: {__html: rocketSvg }}),
```

Comment:
we might want to vertically align Launchpad a bit and add some more spacing

Reply:
Do you mean adding more space between the icon and the "Launchpad"?

Reply:
![screen shot 2017-08-28 at 3 22 06 pm](https://user-images.githubusercontent.com/1562008/29773131-af3188fe-8c04-11e7-8234-83c14e0482b0.png)

Done